### PR TITLE
Bump ace editor to 1.2.5

### PR DESCRIPTION
**Pull request by icewind1991**
The current version is ~3 years old.

cc @LukasReschke @rullzer

### CODE REVIEW

**Comment by LukasReschke** [comment]
Tested 👍

**Comment by rullzer** [comment]
My console tells me:

`../apps/files_texteditor/js/core/vendor/ace-builds/src-min-noconflict/theme-clouds.js`

can't be loaded.

**Comment by rullzer** [comment]
(throws a 404)

**Comment by rullzer** [comment]
Ah we don't have the minified sources anymore it seems

**Comment by icewind1991** [comment]
fixed

File: js/editor.js [hunk 1]
```diff
@@ -508,7 +508,7 @@ var Files_Texteditor = {
 	},
 
 	/**
-	 * Send the new file data back to the server
+	 * Send the new file data back to the serverc
```
Comment (nickvergessen) [inline js/editor.js]: 🙈

**Comment by nickvergessen** [comment]
I think @LukasReschke change will get lost when the next dev runs bower and commits?

**Comment by nickvergessen** [comment]
Last commit is not signed 🙊

**Comment by LukasReschke** [comment]
> I think @LukasReschke change will get lost when the next dev runs bower and commits?

Luckily that library here gets never updated as seen above 🙈

**Comment by icewind1991** [comment]
I added @LukasReschke's patch as file next to the bower.json to make it a bit easier to remember to apply it

**Pull request merged**